Milstone5/index.js

```diff
@@ -0,0 +1,47 @@
+const fibBtn = document.getElementById("fibCaller");
+fibBtn.addEventListener("click", spinnerOn);
+fibBtn.addEventListener("click", fetchData);
+let fibInput = document.getElementById("input");
+const largerThan = document.getElementById("larger-than");
+const serverErr = document.getElementById("life-meaning");
+
+function spinnerOn() {
+    document.getElementById("spinner").innerHTML = `<div class="spinner-border" role="status">
+  <span class="visually-hidden">Loading...</span>
+</div>`;
+}
+
+function spinnerOff() {
+    document.getElementById("spinner").innerHTML = ``;
+}
+
+
+function fetchData() {
+    let url=`http://localhost:5050/fibonacci/${fibInput.value}`;
```

Comment:
You should define the server url as a const before the function, that way it is only created once, and then you just add the number accordingly